labelbox/client.py

```diff
@@ -52,7 +53,10 @@ def __init__(self,
             api_key = os.environ[_LABELBOX_API_KEY]
         self.api_key = api_key
 
-        logger.info("Initializing Labelbox client at '%s'", endpoint)
+        if verbose:
+            logging.basicConfig(level=logging.INFO)
```

Comment:
found some interesting stuff on this one! so `logger.setLevel` will update the LOGGER's level, but because the logger is independent of the HANDLER, even by updating the logger itself, we will still not get any output.

default handler is set to warning and this occurs when we call absolutely no handler (handler of last resort, which has level set to warning)

`logging.basicConfig` is actually called whenever we do not have any set handler, so it defaults to warning. by updating this to logging.INFO, we are updating the default handler's level to INFO instead of WARNING

i suppose if we did not want to update the actual default handler's level, we can always go back to the idea of adding in a handler ourselves

Reply:
additional information provided [here](https://stackoverflow.com/questions/43109355/logging-setlevel-is-being-ignored) and here and here